qiskit_experiments/data_processing/nodes.py

```diff
@@ -272,19 +243,20 @@ def train(self, data: List[Any]):
         qubit so that future data points can be projected onto the axis.
 
         Args:
-            data: A list of datums. Each datum will be converted to a 2D array.
+            data: A full array of IQ data to be trained.
         """
         if data is None:
             return
 
-        data, _ = self._format_data(data)
+        # TODO do not remove standard error. Currently svd is not supported.
+        data = unp.nominal_values(self._format_data(data))
```

Comment:
What is preventing us from doing this?

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/481/commits/09403306e8762d97c155723191deded0f38601ab
Is this expected behavior, right? We don't need to consider the sigma of min/max values because this is not a data processing but just a scaling.

Reply:
I think this is okay for now.